Colours_And_Fonts_style_v1.css

```diff
@@ -198,17 +288,24 @@ ul li:hover ul {
 
 .content{
 	box-sizing: border-box;
-	background-color: #219ebc;
+	background-color: #FFFFFF;

```

Comment:
I changed the colour from a dark blue to white so people can see the text more cearly.